src/MongoDB.Driver.Core/Core/Operations/BulkUnmixedWriteOperationBase.cs

```diff
@@ -157,7 +157,10 @@ public async Task<BulkWriteOperationResult> ExecuteAsync(IWriteBinding binding,
         protected abstract bool RequestHasHint(TWriteRequest request);
 
         // private methods
-        private BulkWriteBatchResult CreateBatchResult(Batch batch, BsonDocument writeCommandResult)
+        private BulkWriteBatchResult CreateBatchResult(
+            Batch batch,
+            BsonDocument writeCommandResult,
+            MongoWriteConcernException writeConcernException)
```

Comment:
Minor stylistic point. When do we keep parameters on the same line of the method definition versus breaking them across multiple lines? I would personally put all three parameters on the same line as the method name.

Reply:
We don't have formalized rule for parameters wrapping, so currently it is mostly done intuitively according to authors preferences, though we have a pending style question to regulate this rule.